src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -481,7 +483,7 @@ internal override bool IsClosureScope(SyntaxNode node)
             return LambdaUtilities.IsClosureScope(node);
         }
 
-        protected override SyntaxNode? FindEnclosingLambdaBody(SyntaxNode? container, SyntaxNode node)
+        protected override SyntaxNode? FindEnclosingLambdaBody(SyntaxNode? container, SyntaxNode? node)
```

Comment:
>?  [](start = 96, length = 2)

`node` parameter should be non-null. It is only possibly assigned null within this method. Better would be to introduce a local `SyntaxNode? current = node;` and replace the usage of `node` with `current` within the body.